### DISCUSSION TITLE
Bot and CI instance missing "CDT" in the name

### DISCUSSION THREAD

**Discussion by jonahgraham** [eclipse-cdt-cloud/cdt-cloud#19] [Q&A] [answered]
Our project's CI instance if called [`cloud`](https://ci.eclipse.org/cloud), the bot is called [`eclipse-cloud-bot`](https://www.npmjs.com/~eclipse-cloud-bot), and the publish location is [`download.eclipse.org/cloud`](https://download.eclipse.org/oomph/archive/download.eclipse/cloud/index.html). 

Before I make request to Webmaster to change `cloud` to `cdt-cloud` in those cases above I wanted to check with fellow members to make sure that is ok. If I don't hear back soon I will raise issue on the [EF HelpDesk](https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/new).

**Comment by sdirix** [comment]
I'm fine with changing it :+1:

**Comment by jonahgraham** [comment, marked as answer]
HelpDesk request made https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1190

**Reply by jonahgraham** [reply]
Turns out the short name of our project in Eclipse Foundation systems was entered as cdt.cloud, but dots are not allowed. Follow the above issue for details. Expect a bunch of small changes like bot names, download locations, etc. 

This does not affect the project brand name that can contain dots: Eclipse CDT.cloud is fine

**Reply by jonahgraham** [reply]
The EF webmaster are fixing this Wednesday from Fred (https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1195#note_695574):

> /cc [@jograham](https://github.com/jograham) please expect some disruptions on Wednesday due to the renaming.
